packages/plugins/build-report/src/xpack.ts

```diff
@@ -367,7 +367,9 @@ export const getXpackPlugin =
                     // Filter the ones that includes the entry name.
                     .filter(
                         (f) => f.includes(name) || (entrypoint.name && f.includes(entrypoint.name)),
-                    )[0];
+                    )
+                    // Only keep JS files.
+                    .filter((f) => getType(f) === 'js')[0];
```

Comment:
To note: if you use `filter` to then return the 1st element, you may want to use `.find` instead

Reply:
Fair

Reply:
This will avoid creating a new array